### PR TITLE
Assorted changes

### PR DESCRIPTION
**Pull request by PubliusScriboniusScholasticus**
These changes are not yet ready because there are some more coming.

### CODE REVIEW

**Comment by nixnull** [comment]
Not sure if it'll send me a notification when you add more commits to this so just @ me when you're done.

File: Email.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
-Of course, a big advantage of email is choice and portability. I and other experienced players use mobile, web, and desktop clients depending on current circumstances and needs. And some players get by exclusively using gmail's web client. So don't worry too much about it.
+Of course, a big advantage of email is choice and portability. I and other experienced players use mobile, web, and desktop clients depending on current circumstances and needs. And some players get by exclusively using Gmail's web client. So don't worry too much about it.
 
 # Mailing List Basics
 
-Agora is played on mailing lists. The mailing lists are set up such that when you send a message to one of the lists, it sends that message to all subscribers. Each list will prefix DIS, BUS, or OFF before new
-messages to the list and Re: before replies to existing messages.
+Agora is played on mailing lists. The mailing lists are set up such that when you send a message to one of the lists, it sends that message to all subscribers. Each list will prefix DIS, BUS, or OFF before new messages to the list and Re: before replies to existing messages.
```
Comment (PubliusScriboniusScholasticus) [inline Email.md]: Does Agora actually add "Re:" to replies? I've not observed this.

Comment (nixnull) [reply Email.md]: I might be wrong there, it might just be people's clients that add it; we can just take it out for now.

**Comment by PubliusScriboniusScholasticus** [comment]
@nixnull This is all the problems I see now.

**Review by nixnull — APPROVED** [review]
Looks good to me, thanks a ton!

**Pull request merged**